.github/workflows/publish_pypi.yml

```diff
@@ -0,0 +1,32 @@
+name: Publish package to PyPi
+
+on:
+    # Triggers the workflow when release is published
```

Comment:
other teams are using release published trigger, so should we use the same or merged to master is fine?

Reply:
I think this is ok since it's an established pattern.

Reply:
Since this is an open source package, I would recommend we just do it the normal industry way of triggering publish from a release, and _remove the push-to-master trigger_.  As long as our workflow only has cutting a release as the only manual step and the rest is handled automatically, then I think it's better to be more normal.

Reply:
I am in favor of making it as **Publish on Release** That is what I am seeing in some example on blog posts etc.
I have asked in SRE channel what is the common practice edX teams follow and will update it accordingly once I get their response.